pyht/client.py

```diff
@@ -283,6 +285,8 @@ def _tts(
         else:
             text = [normalize(x) for x in text]
         text = ensure_sentence_end(text)
+        if self._advanced.remove_ssml_tags:
```

Comment:
Seems you should probably do this before the official `normalize` happens.

Reply:
Done